admin/model_cloud_cluster.go

```diff
@@ -20,7 +20,7 @@ type CloudCluster struct {
 	// Flag that indicates whether the cluster can perform backups. If set to `true`, the cluster can perform backups. You must set this value to `true` for NVMe clusters. Backup uses Cloud Backups for dedicated clusters and Shared Cluster Backups for tenant clusters. If set to `false`, the cluster doesn't use MongoDB Cloud backups.
 	// Read only field.
 	BackupEnabled *bool `json:"backupEnabled,omitempty"`
-	// Unique 24-hexadecimal character string that identifies the cluster.
+	// Unique 24-hexadecimal character string that identifies the cluster. Each ``clusterId`` is used only once across all MongoDB Cloud deployments.
```

Comment:
I wonder why there is a `` here?

Reply:
Change was introduced in OpenAPI.
SDK uses OpenAPI as source for generation of the comments.